### PR TITLE
Fix Coveralls reporting

### PR DESCRIPTION
**Pull request by mprazz**
**Proposed changes**:
- Since Feb/Mar we upload several reports to coveralls, and hence we must tell coveralls we are done reporting :) 

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/main/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: .github/workflows/continous-integration.yml [hunk 1]
```diff
@@ -311,6 +311,20 @@ jobs:
...
 
+  complete_coverage:
+    name: Complete coverage
+    runs-on: ubuntu-20.04
+    needs: [ test, changes ]
+    if: needs.changes.outputs.backend == 'true'
+
+    steps:
+      - name: Tell coveralls we're finished reporting
+        uses: coverallsapp/github-action@master
```
Comment (wochinge) [inline .github/workflows/continous-integration.yml:322]: should we pin a specific version / hash to avoid that they potentially break our CI with changes?

**Review by wochinge — APPROVED** [review]

**Comment by wochinge** [comment]
Thanks for the blazing fast fix 🙌🏻

File: .github/workflows/continous-integration.yml (hunk 1 above)
Comment (mprazz) [reply .github/workflows/continous-integration.yml:322]: We had `master` on `rasa-x` for ages without problems :thinking: plus, their past release was september last year -- coveralls seems to not be that well maintained tbh

**Comment by wochinge** [comment]
sweet. [It worked](https://coveralls.io/builds/41071759) 🙌🏻

**Comment by mprazz** [comment]
NOICE! Was on a meeting, thanks for cleaning up that dummy commit @wochinge :pray:

**Pull request merged**